Test/Install-WhiskeyDotNetTool.Tests.ps1

```diff
@@ -305,16 +387,25 @@ try
             GivenWorkingDirectory 'app'
             GivenGlobalJsonSdkVersion '1.0.1' -Directory $workingDirectory
             GivenGlobalJsonSdkVersion '2.1.505' -Directory $TestDrive.FullName
+            GivenGlobalDotNetHasValidVersion -Version '1.1.11'
             GivenVersion '1.1.11'
             WhenInstallingDotNetTool
-            ThenReturnedValidDotNetPath
+            ThenReturnedDotNetExecutable
             ThenDotNetNotLocallyInstalled
-            ThenGlobalJsonVersion '1.1.11' -Directory $workingDirectory
-            ThenGlobalJsonVersion '2.1.505' -Directory $TestDrive.FullName
+        }
+    }
+
+    Describe 'Install-WhiskeyDotNetTool.when installing DotNet and patch lower is already installed' {
+        It 'should use global.json''s version' {
+            Init
+            GivenGlobalDotNetInstalled '2.1.514'
+            GivenGlobalJsonRollForwardAndSdkVersion -Version '2.1.500' -RollForward Patch
+            GivenGlobalDotNetHasValidVersion -Version '2.1.514'
+            WhenInstallingDotNetTool

```

Comment:
This test has no assertions. It should be asserting that:
* The function returns `'dotnet'`.
* That `Install-WhiskeyDotNetSdk` is never called.